src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1473,7 +1473,7 @@ internal static bool IsCustomTaskType(this NamedTypeSymbol type, [NotNullWhen(tr
                         && attr.CommonConstructorArguments.Length == 1
                         && attr.CommonConstructorArguments[0].Kind == TypedConstantKind.Type)
                     {
-                        builderArgument = attr.CommonConstructorArguments[0].ValueInternal;
```

Comment:
Was this incidental to the change, or a result of the NullableWalker change?

Reply:
Without this, we can't bootstrap. The compiler would complain that `buildArgument` is `null` upon exit.
But we know it's not null in this case, because we checked the `.Kind` above.